docs/api/source/index.rst

```diff
@@ -13,7 +13,24 @@ For Solid Servers that support notifications, they are sent when a resource is
 created, updated, deleted, or when a child resource is added to or removed from
 a container.
 
-**This library currently only supports notifications sent via WebSockets.**
+.. admonition:: Supported Protocols
+   :class: caution
+
+   This library currently only supports an `early version of WebSocket
+   notifications
+   <https://docs.inrupt.com/ess/1.1/services/service-websocket/>`__, which
```

Comment:
So ... I might be a bit more explicit in that it supports ESS WebSocket notifications. 
And omit all about forming the basis of the Solid Notification Protocol ...

Reply:
I'll defer to @acoburn here.

Reply:
Agree with Kay here. Let's just say "This library currently only supports ESS Websocket notifications."